tests/Batch/BatchTest.php

```diff
@@ -168,17 +171,16 @@ public function testReverseShouldMadeCorrectTasksArrayToComputeWithManyCoordinat
     }
 
     /**
-     * @expectedException League\Geotools\Exception\InvalidArgumentException
-     * @expectedExceptionMessage The argument should be a Coordinate instance or an array of Coordinate instances to reverse.
      * @dataProvider coordinatesProvider
      */
     public function testReverseShouldThrowInvalidArgumentException($coordinates)
     {
+        $this->expectException(InvalidArgumentException::class);
         $batch = new TestableBatch($this->geocoder);
         $batch->reverse($coordinates);
     }
 
-    public function coordinatesProvider()
+    public static function coordinatesProvider(): array
```

Comment:
Why method is declared as static ?

Reply:
Because it's a standard approach to writing data providers.